rmgpy/rmg/main.py

```diff
@@ -709,14 +734,14 @@ def updateReactionThresholdAndReactFlags(self, rxnSysUnimolecularThreshold=None,
         numCoreSpecies = len(self.reactionModel.core.species)
         prevNumCoreSpecies = len(self.unimolecularReact)
         stale = True if numCoreSpecies > prevNumCoreSpecies else False
-        
-        
+
+
```

Comment:
can you remove those empty line changes from the commit? there are several occurrance down the file as well